.nvmrc

```diff
@@ -0,0 +1 @@
+16.16.0
```

Comment:
any reason not to update it to node 18?

Reply:
Yes. Upgrading node means also updating React. Upgrading react is a non trivial piece of work and very time intensive for me personally with little react experience.